pandas/tests/extension/test_integer.py

```diff
@@ -143,6 +143,12 @@ def test_error(self, data, all_arithmetic_operators):
         # other specific errors tested in the integer array specific tests
         pass
 
+    @pytest.mark.xfail(reason="EA is listified. GH-22922", strict=True)
```

Comment:
So, here's what's happening for IntegerArray with missing values,

Because of #22922, the ExtensionArray is listified, so we do `IntegerArray + List`. Then, in `IntegerArray.__add__` we do `np.asarray(other)`. Since we have missing values, we naturally end up with a float ndarray.

Fixing #22922 will fix this specific issue, but doesn't handle the general problem of `IntegerArray + List`. i.e. is this a bug?

```python
In [3]: arr = pd.Series([1, None, 3], dtype='Int8').values

In [4]: arr
Out[4]: IntegerArray([1, nan, 3], dtype='Int8')

In [5]: arr + list(arr)
Out[5]: array([ 2., nan,  6.])
```

Should `Out[5]` be an `IntegerArray([2, nan, 6], dtype='Int8')`? If so, I'll open a new issue.

Reply:
yes i think you are right

not sure this is hitting the right path

Reply:
https://github.com/pandas-dev/pandas/issues/22962